src/backend/dx11/src/device.rs

```diff
@@ -457,19 +457,34 @@ impl Device {
                 }
             }
             image::ViewKind::D2 => {
-                desc.ViewDimension = d3dcommon::D3D11_SRV_DIMENSION_TEXTURE2D;
-                *unsafe { desc.u.Texture2D_mut() } = d3d11::D3D11_TEX2D_SRV {
-                    MostDetailedMip,
-                    MipLevels,
+                if info.kind.num_samples() > 1 {
```

Comment:
we should probably lift that condition to the match arm, i.e.
```rust
image::ViewKind::D2 if info.kind.num_samples() > 1 => {...}
image::ViewKind::D2 => {...}
```
The benefit is that all match arms become very similar and nested in the same way